.github/workflows/build_BE.yml

```diff
@@ -90,8 +90,10 @@ jobs:
       - name: Creating Lint + Bug Reports
         run: |
           rm -f ../*_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../lint_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../bug_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../BE_lint_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../BE_bug_report.txt
+          npx eslint . -f node_modules/eslint-html-reporter/reporter.js -o ../FE_bug_report.html
+          npx standard | npx standard-reporter --json > ../FE_lint_report.json
```

Comment:
@jkim10 The error you're getting is related to `working_directory` which is defaulted to `backend` in line 10